distribution/errors.go

```diff
@@ -112,6 +112,23 @@ func TranslatePullError(err error, ref reference.Named) error {
 	return errdefs.Unknown(err)
 }
 
+func isNotFound(err error) bool {
+	switch v := err.(type) {
+	case errcode.Errors:
+		for _, e := range v {
```

Comment:
Should it continue looping, or pick the first error? I recall we had some code similar in https://github.com/moby/moby/blob/master/errdefs/http_helpers.go#L157 which was based on code from the distribution packages

Reply:
I don't see why we wouldn't look at all the returned errors in this case.